service/history/timerQueueActiveTaskExecutor.go

```diff
@@ -219,7 +219,7 @@ Loop:
 			break Loop
 		}
 
-		timeoutFailure := failure.NewTimeoutFailure(timerSequenceID.timerType)
+		timeoutFailure := failure.NewTimeoutFailure("ActivityTask timeout", timerSequenceID.timerType)
```

Comment:
Here is the thing: `TimeoutType` is not immutable and it is getting changed on line 238 bellow. Message should reflect it. I don't see good solution for it and I would prefer not to have `TimeoutType` in the message.

Reply:
good point